ManatiCR/vitaminau
# VIT-65: Help text update in beneficiaries section.

# Task [VIT-65](https://manati.atlassian.net/browse/VIT-65)
Update help text in the Economic activity field of the beneficiaries section.

# Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the followings urls: 
  - http://vitaminau.docker/form/webform-entrepreneurship-support
  - http://vitaminau.docker/form/profesional-practice
  - http://vitaminau.docker/form/laboratory-services
  - http://vitaminau.docker/form/technical-services
  - http://vitaminau.docker/form/webform-licensed-technologies
  - http://vitaminau.docker/form/webform-installations-use
- Verify that the description of the "Sector de actividad económica" field in the beneficiaries section is "Si no se encuentra seguro de la categoría, puede revisar a detalle el Código CIIU https://unstats.un.org/unsd/publication/SeriesM/seriesm_4rev4s.pdf."

# CA
- The "Sector de actividad económica" field must have the following description "Si no se encuentra seguro de la categoría, puede revisar a detalle el Código CIIU https://unstats.un.org/unsd/publication/SeriesM/seriesm_4rev4s.pdf" in the following forms:
  - Apoyo al emprendimiento
  - Práctica profesional
  - Servicios de laboratorio
  - Servicios técnicos
  - Tecnologías licenciables
  - Uso de instalaciones
